### DISCUSSION TITLE
Support for audiobook series

### DISCUSSION THREAD

**Discussion by MakD** [PaulWoitaschek/Voice#1748] [Ideas]
Is it possible to add or implement series in the main page, with the other audiobooks inside them, as in, the home page shows the Harry Potter series, and upon clicking, shows all the available audiobooks. Could be a life-changing feature for those who like to collect audiobook series, like me. Also, some metadata providers would be great.

**Comment by MakD** [comment]
@PaulWoitaschek

**Comment by mesonium** [comment]
Sounds like https://github.com/PaulWoitaschek/Voice/discussions/1314

**Reply by MakD** [reply]
Any way to implement the same
